src/splits.cpp

```diff
@@ -2,35 +2,34 @@
 
 // [[Rcpp::export]]
 IntegerVector splits(IntegerMatrix dm, IntegerVector community, int nd, int max_split) {
-  IntegerVector ret(dm.ncol());
-  IntegerVector com_name = sort_unique(community);
-  int nc = com_name.size();
-  IntegerMatrix com_found(nc, nd);
-  IntegerVector mid;
+    IntegerVector ret(dm.ncol());
```

Comment:
what still uses this?

Reply:
and is there a way to replace with a call to a redistmetrics dynlib?

Reply:
Some short burst scorers use it. Can look into how dynlib works for this.